safeyaml.py

```diff
@@ -576,48 +673,101 @@ def parse_bareword(buf, pos, output, options):
+    raise Bareword(
+        buf, pos,
+        "The parser doesn't know how to parse anymore and has given up. Use less barewords"
+    )
 
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description="SafeYAML Linter, checks (or formats) a YAML file for common ambiguities")
-
-    parser.add_argument("file", nargs="*", default=None, help="filename to read, without will read from stdin")
-    parser.add_argument("--fix",  action='store_true', default=False, help="ask the parser to hog wild")
-    parser.add_argument("--fix-unquoted",  action='store_true', default=False, help="ask the parser to try its best to parse unquoted strings/barewords")
-    parser.add_argument("--fix-nodent",  action='store_true', default=False, help="fix lists to nest inside maps with same indent")
-    parser.add_argument("--fix-nospace",  action='store_true', default=False, help="fix map keys not to have ' ' after a ':'")
-    parser.add_argument("--force-string-keys",  action='store_true', default=False, help="quote every bareword")
-    parser.add_argument("--force-commas",  action='store_true', default=False, help="trailing commas")
-    parser.add_argument("--quiet", action='store_true', default=False, help="don't print cleaned file")
-    parser.add_argument("--in-place", action='store_true', default=False, help="edit file")
-
-    parser.add_argument("--json", action='store_true', default=False, help="output json instead of yaml")
-
-    args = parser.parse_args() # will only return when action is given
-    
+    parser = argparse.ArgumentParser(
+        description=
+        "SafeYAML Linter, checks (or formats) a YAML file for common ambiguities"
+    )
+
+    parser.add_argument(
+        "file",
+        nargs="*",
+        default=None,
+        help="filename to read, without will read from stdin")
+    parser.add_argument(
+        "--fix",
+        action='store_true',
+        default=False,
+        help="ask the parser to hog wild")
+    parser.add_argument(
+        "--fix-unquoted",
+        action='store_true',
+        default=False,
+        help=
+        "ask the parser to try its best to parse unquoted strings/barewords")
+    parser.add_argument(
+        "--fix-nodent",
+        action='store_true',
+        default=False,
+        help="fix lists to nest inside maps with same indent")
+    parser.add_argument(
+        "--fix-nospace",
+        action='store_true',
+        default=False,
+        help="fix map keys not to have ' ' after a ':'")
+    parser.add_argument(
+        "--force-string-keys",
+        action='store_true',
+        default=False,
+        help="quote every bareword")
+    parser.add_argument(
+        "--force-commas",
+        action='store_true',
+        default=False,
+        help="trailing commas")
+    parser.add_argument(
+        "--quiet",
+        action='store_true',
+        default=False,
+        help="don't print cleaned file")
+    parser.add_argument(
```

Comment:
This should maybe be split to keep it line with the other arguments?

Reply:
I tried, but yapf just pops it back again. Shrug!